### DISCUSSION TITLE
Username passthrough to docker backend

### DISCUSSION THREAD

**Discussion by bmeekhof** [ContainerSSH/ContainerSSH#600] [Questions]
Hi, I'm running containerssh with the docker backend, OIDC device auth, and an authz webhook server.  I'm able to authenticate and get successful request/response from my webhook server but regardless of what I set authenticatedUsername to the backend always spins up containers as user root (my containerssh server is running is root).  Is there a way to pass the authenticated username through to the container the docker backend spins up?  

I realize there is a static 'user:' param that can be used but this isn't quite what I'm looking for.  If I login as a username, I want my name in the container to be that username.

**Comment by tsipinakis** [comment]
The username you see on linux is just an alias for an id (usually stored in `/etc/passwd`, or fetched through LDAP). You cannot use a user/username unless the user exists there. So if you want to do that you'll either need to declare all the users in `/etc/passwd` that are going to log in, or use LDAP with a service like sssd to fetch the user information. Then you can set the user the same way from the configuration server.
